tests/wpt/web-platform-tests/css/css-counter-styles/broken-symbols.htm

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<title>CSS Test: invalid counter-style symbols</title>
+<link rel="author" title="Anthony Ramine" href="mailto:n.oxyde@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-counter-styles-3/#typedef-symbol">
+<link rel="match" href="broken-symbols-ref.htm">
+<style type="text/css">
+  @counter-style a {
+    system: alphabetic;
+    symbols: ⓐ inherit;
+  }
+</style>
+<ol style="list-style-type: a"><li>foo</ol>
```

Comment:
Maybe mention the pass condition instead of `foo`? Like "Should see a normal list bullet" or something?

Reply:
Done.